tests/models/bert/test_modeling_bert.py

```diff
@@ -463,7 +463,6 @@ class BertModelTest(ModelTesterMixin, GenerationTesterMixin, PipelineTesterMixin
         if is_torch_available()
         else {}
     )
-    fx_compatible = True
```

Comment:
FX compatibility is broken for BERT, ELECTRA, MobileBERT, and RoBERTa. I'm not sure how critical this is. As far as I understand, we dropped FX compatibility for text models previously by adding `**kwargs`.

It's a bit strange that FX compatibility is not broken for CLIP, even though it also has the `@can_return_tuple` decorator.